src/formatDate.js

```diff
@@ -8,7 +8,34 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldSeparator = fromFormat.slice(3);
```

Comment:
The `slice` method here is being used incorrectly to get the separator. Since `fromFormat` is an array, you should be using an index to access the separator instead of `slice`. Use `fromFormat[3]` to get the separator.